utam-compiler/src/main/java/utam/compiler/UtamCompilerIntermittentError.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.compiler;
+
+import com.fasterxml.jackson.core.JsonLocation;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.function.Supplier;
+import utam.compiler.helpers.TranslationContext;
+
+/**
+ * This exception is thrown from deserializer when we do not know the context. Then we catch it and
+ * process
+ *
+ * @author elizaveta.ivanova
+ * @since 238
+ */
+public class UtamCompilerIntermittentError extends RuntimeException {
```

Comment:
I'd call this `Intermediate` or `Interstitial` as opposed to `Intermittent`. The latter seems to imply "it happens sometimes."